packages/api/src/controllers/taskController.js

```diff
@@ -1098,3 +1272,7 @@ async function filterOutDeletedManagementPlans(data, req) {
     validPlantingMangementPlans.includes(planting_management_plan_id),
   );
 }
+
+const flattenInternalIdentifier = (animalOrBatch) => {
+  animalOrBatch.internal_identifier = animalOrBatch.internal_identifier.internal_identifier;
```

Comment:
Why do we have nested internal identifiers? 🤔

Reply:
Ah, I see!! I know this is pre-existing but I think we should change the name either on the relation or the column to make it less confusing